x/oracle/keeper/oracle_test.go

```diff
@@ -33,7 +35,7 @@ func TestOracleTestSuite(t *testing.T) {
 }
 
 func (suite *oracleTestSuite) TestRegisterOracle() {
-	err := suite.BankKeeper.AddCoins(suite.Ctx, oracle1, fundForOracle)
+	err := simapp.FundAccount(suite.BankKeeper, suite.Ctx, oracle1, fundForOracle)
```

Comment:
My guess is that `simapp.FuncAccount` can also change, and if it does, we'll need a lot of code changes.
How about implementing `FundAccount` in `Testsuite` struct and calling it in the form of `suite.FundAccount`?
We will be able to respond more flexibly to these changes.

Reply:
As you suggested, I made a `FundAccount()` in `testsuite`, and replaced all `simapp.FundAccount()` to the func of `testsuite`. It's good idea to make independent to cosmos-sdk, if possible. Thx!